### PR TITLE
Add linux cuda 11.7 workflows

### PR DESCRIPTION
**Pull request by ptrblck**
Ref https://github.com/pytorch/builder/issues/1042

CC @atalman

### CODE REVIEW

**Comment by atalman** [comment]
@ptrblck Please rebase this should fix docker-builds

File: .github/workflows/docker-builds.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
           - docker-image-name: pytorch-linux-bionic-cuda10.2-cudnn7-py3.9-gcc7
           - docker-image-name: pytorch-linux-bionic-cuda11.3-cudnn8-py3-clang9
           - docker-image-name: pytorch-linux-bionic-cuda11.6-cudnn8-py3-gcc7
+          - docker-image-name: pytorch-linux-bionic-cuda11.7-cudnn8-py3-gcc7
           - docker-image-name: pytorch-linux-bionic-py3.7-clang9
```
[the remark this answers is not in the document]
Comment (ptrblck) [reply .github/workflows/docker-builds.yml:40]: Added it! :)

**Review by atalman — APPROVED** [review]
LGTM

**Pull request closed without merging**